test/geometry/GeoParams.test.cc

```diff
@@ -38,10 +38,22 @@ class GeoParamsHostTest : public GeoParamsTest
 TEST_F(GeoParamsHostTest, accessors)
 {
     const auto& geom = *(this->params());
-    EXPECT_EQ(2, geom.num_volumes());
-    EXPECT_EQ(2, geom.max_depth());
-    EXPECT_EQ("Detector", geom.id_to_label(VolumeId{0}));
-    EXPECT_EQ("World", geom.id_to_label(VolumeId{1}));
+    EXPECT_EQ(11, geom.num_volumes());
+    EXPECT_EQ(4, geom.max_depth());
+    EXPECT_EQ("Envelope",
+              geom.id_to_label(VolumeId{
+                  static_cast<unsigned int>(geom.num_volumes()) - 2}));
+    EXPECT_EQ("World",
+              geom.id_to_label(VolumeId{
+                  static_cast<unsigned int>(geom.num_volumes()) - 1}));
+
+    // print geometry information from CPU
+    unsigned int nvols = geom.num_volumes();
+    for (unsigned int i = 0; i < nvols; ++i)
+    {
+        std::cout << "id " << i << ":  " << geom.id_to_label(VolumeId{i})
+                  << std::endl;
+    }
```

Comment:
Can you convert this into something that tests (`EXPECT_EQ`) rather than just printing to screen? Or remove if it's just debugging code.

Reply:
It was just debugging code.  Removed.